test/directory.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:io';
+
+import 'package:test/test.dart';
+import 'package:dartx/dartx_io.dart';
+
+void main() {
+  group('Directory', () {
+    test('contains', () async {
+      final thisDir = Directory.current;
```

Comment:
Please use a temp directory when you create files during a test

Reply:
see new commit